### PR TITLE
SpellCheckBear.py: Added Documentation

### PR DESCRIPTION
**Pull request by sounak98**
SpellCheckBear.py:	Added Documentation

The documentation advertises the application of this Bear

Fixes https://github.com/coala/coala-bears/issues/1047

### CODE REVIEW

**Comment by nemaniarjun** [comment]
Please run coala in your home directory before committing.

**Comment by Mixih** [comment]
You need to squash your commits into one message (see https://docs.coala.io/en/latest/Developers/Git_Basics.html#squashing-your-commits)

**Comment by aptrishu** [comment]
Fix the gitmate issues. and run `coala` locally then make a PR it'd be helpful. :)

File: bears/natural_language/SpellCheckBear.py [hunk 1]
```diff
@@ -10,7 +10,14 @@ class SpellCheckBear:
...
 
-    See <https://pypi.python.org/pypi/scspell3k> for more information.
+    scspell is a spell checker for source code.
+    When applied to a code written in most popular programming languages
+    while using most typical naming conventions, this algorithm will
+    usually catch many errors without an annoying false positive rate.
+
+    In an effort to catch more spelling errors, scspell is able to
+    check each file against a set of dictionary words selected
+    specifically for that file.
```
Comment (sounak98) [inline bears/natural_language/SpellCheckBear.py]: Added. Sorry, I missed out the "add-the-link-to-its-package" part in the previous comment. My bad.

**Comment by tulphoon** [comment]
ack 9bd9099

**Comment by tulphoon** [comment]
@rultor merge

**Pull request merged**